HighwayPatrol.h

```diff
@@ -1,11 +1,14 @@
 #pragma once 
 
 #include "Vehicle.h"
+#include "Highway.h"
 
 struct HighwayPatrol : Vehicle 
 {
     HighwayPatrol();
+    ~HighwayPatrol() override;
+
```

Comment:
Don't forget to implement the other 2 functions that go with the destructor!
- [ ] revise